src/attribution/onCLS.ts

```diff
@@ -25,11 +25,11 @@ import {
 } from '../types.js';
 
 const getLargestLayoutShiftEntry = (entries: LayoutShift[]) => {
-  return entries.reduce((a, b) => (a && a.value > b.value ? a : b));
+  return entries.reduce((a, b) => (a?.value > b.value ? a : b));
 };
 
 const getLargestLayoutShiftSource = (sources: LayoutShiftAttribution[]) => {
-  return sources.find((s) => s.node && s.node.nodeType === 1) || sources[0];
+  return sources.find((s) => s?.node?.nodeType === 1) || sources[0];
```

Comment:
```suggestion
  return sources.find((s) => s.node?.nodeType === 1) || sources[0];
```
yah?

Reply:
Yeah, I don't think `s` will ever be undefined.